### PR TITLE
Mention EnvironmentConfiguration in the list of configuration sources

### PR DESCRIPTION
**Pull request by obfischer**

### CODE REVIEW

File: src/site/xdoc/userguide/overview.xml [hunk 1]
```diff
@@ -46,6 +46,10 @@
         JNDIConfiguration, you are sheltered from the mechanics of actually retrieving the
         configuration values. These various sources include:
         <ul>
+          <li>
+              <strong>EnvironmentConfiguration</strong>
+              Reads the plattform specific environment variables.
```
Comment (sebbASF) [inline src/site/xdoc/userguide/overview.xml:51]: s/plattform/platform/

**Comment by obfischer** [comment]
👍 🙇🏻‍♂️

File: src/site/xdoc/userguide/overview.xml (hunk 1 above)
Comment (obfischer) [reply src/site/xdoc/userguide/overview.xml:51]: Sorry for the typo. @garydgregory merged it already. Should I create a new PR?

Comment (garydgregory) [reply src/site/xdoc/userguide/overview.xml:51]: Yep.

**Pull request merged**